app/src/main/java/org/mifos/mobile/ui/fragments/LoanAccountsDetailFragment.kt

```diff
@@ -153,67 +114,94 @@ class LoanAccountsDetailFragment : BaseFragment(), LoanAccountsDetailView {
      *
      * @param loanWithAssociations object containing details of each loan account,
      */
-    fun showDetails(loanWithAssociations: LoanWithAssociations?) {
-        tvOutstandingBalanceName?.text = resources.getString(R.string.string_and_string,
+    private fun showDetails(loanWithAssociations: LoanWithAssociations?) {
+        binding.tvOutstandingBalance.text = resources.getString(R.string.string_and_string,
                 loanWithAssociations?.summary?.currency?.displaySymbol, CurrencyUtil.formatCurrency(activity,
                 loanWithAssociations?.summary?.totalOutstanding))
         if (loanWithAssociations?.repaymentSchedule?.periods != null) for ((_, _, dueDate, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, totalDueForPeriod) in loanWithAssociations.repaymentSchedule?.periods!!) {
             if (dueDate == loanWithAssociations.summary?.getOverdueSinceDate()) {
-                tvNextInstallmentName?.text = resources.getString(R.string.string_and_string,
+                binding.tvNextInstallment.text = resources.getString(R.string.string_and_string,
                         loanWithAssociations.summary?.currency?.displaySymbol,
                         CurrencyUtil.formatCurrency(activity,
                                 totalDueForPeriod))
                 break
             } else if (loanWithAssociations.summary?.getOverdueSinceDate() == null) {
-                tvNextInstallmentName?.setText(R.string.not_available)
+                binding.tvNextInstallment.setText(R.string.not_available)
             }
         }
-        tvAccountNumberName?.text = loanWithAssociations?.accountNo
-        tvLoanTypeName?.text = loanWithAssociations?.loanType?.value
-        tvCurrencyName?.text = loanWithAssociations?.summary?.currency?.code
+        binding.tvAccountNumber.text = loanWithAssociations?.accountNo
+        binding.tvLoanType.text = loanWithAssociations?.loanType?.value
+        binding.tvCurrency.text = loanWithAssociations?.summary?.currency?.code
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
```

Comment:
I believe you could use the `with` here as well, so in any place when you need a `binding` instance for more than 2 items I recommend you to use this function. Do the same in all other open PRs as well, please 😊 

cc @PratyushSingh07